packages/gamut/src/Box/Box.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  space,
+  layout,
+  border,
+  color,
+  shadow,
+  positioning,
+  background,
+  typography,
+  flex,
+  grid,
+} from '@codecademy/gamut-styles';
+import { compose, HandlerProps } from '@codecademy/gamut-system';
+import styled from '@emotion/styled';
+
+const boxStyles = compose(
+  space,
+  layout,
+  border,
+  color,
+  shadow,
+  positioning,
+  background,
+  typography,
+  flex,
+  grid
+);
+
+export type BoxStyles = HandlerProps<typeof boxStyles>;
+export interface BoxProps extends BoxStyles {}
```

Comment:
I forget, does having the interface directly extend the `HandlerProps` work?

```suggestion
export interface BoxProps extends HandlerProps<typeof boxStyles> {}
```

Reply:
Yeah :(

It's dumb but it reduces the size by around 60% by preventing `styled.x` generic copying the props.

Reply:
Good use case for a custom lint rule!